README.rst

```diff
@@ -169,4 +169,7 @@ See the individual files for license details.
    :target: https://api.reuse.software/info/github.com/pypsa/atlite
 .. |black| image:: https://img.shields.io/badge/code%20style-black-000000.svg
    :target: https://github.com/psf/black
-   :alt: Code style: black
\ No newline at end of file
+   :alt: Code style: black
+.. |pre-commit.ci| image:: https://results.pre-commit.ci/badge/github/euronion/atlite/master.svg
+   :target: https://results.pre-commit.ci/latest/github/euronion/atlite/master
+   :alt: pre-commit.ci status
```

Comment:
```rst
.. |pre-commit.ci| image:: https://results.pre-commit.ci/badge/github/PyPSA/atlite/master.svg
   :target: https://results.pre-commit.ci/latest/github/PyPSA/atlite/master
   :alt: pre-commit.ci status
```

Reply:
already done in masterr, sorry, I was a bit fast here...